static/styles/compose.css

```diff
@@ -184,6 +184,27 @@
     margin: auto;
 }
 
+#compose_top_right {
+    margin: 0 18px 4px auto;
+    position: absolute;
+    right: 0;
+
+    a {
+        color: hsl(0, 0%, 0%);
+        font-size: 20px;
+        opacity: 0.7;
+        text-decoration: none;
+
+        &:hover {
+            opacity: 1;
+        }
+    }
+}
```

Comment:
The hover behavior seems to not be working when testing this, both for this element and the "X".  A few other notes on the icon:
* It seems a bit misaligned with the "X", and might need a small vertical alignment adjustment.
* We should scale icons with %, not a number of `font-size` pixels, since that will work better when support multiple font sizes.
*

Reply:
Checking this one.

Reply:
1. Fixing this one
2. Actually the close `x` has the font size of 20px, so used that value as if we have different values, it will not look better. If we need to change it to percent, a seprate PR changing this for both will be good.
3. Increasing font weight will do the job, so changed that.

Reply:
OK, looks great.